### PR TITLE
Cleaning up breaking of attributes on generic type parameters

### LINKED ISSUE #291: Improve formatting of attribute in generics
```c#
private static BindParser<T> MakeTypeConverterConverter<[DynamicallyAccessedMembers(
        DynamicallyAccessedMemberTypes.All)]
    T>()
// could maybe look like this, but it still seems off
private static BindParser<T> MakeTypeConverterConverter<
    [DynamicallyAccessedMembers(DynamicallyAccessedMemberTypes.All)]
    T
>()
```

### PR DESCRIPTION
**Pull request by belav**
closes #291

### CODE REVIEW

File: Src/CSharpier.Tests/TestFiles/AttributeList/AttributeLists.cst [hunk 1]
```diff
@@ -66,6 +66,19 @@ class ClassName
     ) {
         return;
     }
+
+    public void MethodWithAttributesOnTypeParameters<[SomeShort] T, K>() { }
+
+    public void MethodWithAttributesOnTypeParameters<
+        [SomeLongAttribute__________________________] T,
```
Comment (shocklateboy92) [inline Src/CSharpier.Tests/TestFiles/AttributeList/AttributeLists.cst:73]: Omg I forgot this was even a thing. C# is a stupidly complicated language...

File: Src/CSharpier.Tests/TestFiles/ObnoxiousEdgeCases/ObnoxiousEdgeCases.cst [hunk 2]
```diff
@@ -57,10 +52,10 @@ class ClassName
     public static IRemoteAuthenticationBuilder<
         RemoteAuthenticationState,
         RemoteUserAccount
-    > AddAccountClaimsPrincipalFactory<[DynamicallyAccessedMembers(
-            DynamicallyAccessedMemberTypes.PublicConstructors
-        )]
-        TAccountClaimsPrincipalFactory>(
+    > AddAccountClaimsPrincipalFactory<
+        [DynamicallyAccessedMembers(DynamicallyAccessedMemberTypes.PublicConstructors)]
```
Comment (shocklateboy92) [inline Src/CSharpier.Tests/TestFiles/ObnoxiousEdgeCases/ObnoxiousEdgeCases.cst:56]: Will this still break if the attribute or its parameters were long enough?

File: Src/CSharpier.Tests/TestFiles/AttributeList/AttributeLists.cst (hunk 1 above)
Comment (belav) [reply Src/CSharpier.Tests/TestFiles/AttributeList/AttributeLists.cst:73]: I don't think I even realized it was a thing until I ran into it while formatting the aspnetcore repo.

File: Src/CSharpier.Tests/TestFiles/ObnoxiousEdgeCases/ObnoxiousEdgeCases.cst (hunk 2 above)
Comment (belav) [reply Src/CSharpier.Tests/TestFiles/ObnoxiousEdgeCases/ObnoxiousEdgeCases.cst:56]: Looks like it, I added a test case that breaks

**Pull request merged**